src/core/qgsmapsettings.h

```diff
@@ -310,6 +310,7 @@ class CORE_EXPORT QgsMapSettings : public QgsTemporalRangeObject
       RenderPartialOutput      = 0x200, //!< Whether to make extra effort to update map image with partially rendered layers (better for interactive map canvas). Added in QGIS 3.0
       RenderPreviewJob         = 0x400, //!< Render is a 'canvas preview' render, and shortcuts should be taken to ensure fast rendering
       RenderBlocking           = 0x800, //!< Render and load remote sources in the same thread to ensure rendering remote sources (svg and images). WARNING: this flag must NEVER be used from GUI based applications (like the main QGIS application) or crashes will result. Only for use in external scripts or QGIS server.
+      Prefetch                 = 0x1000, //!< The render is a prefetched image, not visible in the current view extent
```

Comment:
This is the same as the existing "RenderPreviewJob" flag

Reply:
I though it was for something else. I will re-use that flag then.